mumd/src/audio.rs

```diff
@@ -291,64 +203,15 @@ impl Audio {
     }
 
     pub fn decode_packet_payload(&self, stream_type: VoiceStreamType, session_id: u32, payload: VoicePacketPayload) {
-        match self.client_streams.lock().unwrap().entry((stream_type, session_id)) {
-            Entry::Occupied(mut entry) => {
-                entry
-                    .get_mut()
-                    .decode_packet(payload, self.output_config.channels as usize);
-            }
-            Entry::Vacant(_) => {
-                warn!("Can't find session id {}", session_id);
-            }
-        }
-    }
-
-    pub fn add_client(&self, session_id: u32) {
```

Comment:
I remove the {add, remove, clear}user from audio, now ClientStreams does that on demand.